Amir's SleepBuddy project.md

```diff
@@ -0,0 +1,54 @@
+---
+name: "@LiricTech"
+project: "The SleepBuddy"
+---
+
+![SleepBuddy's logo](https://user-images.githubusercontent.com/108364980/209412196-7229aa15-0fad-49da-9cc0-d680d082f702.png)
+
+
+# The SleepBuddy
+
+## 💁‍♂️ Summary:
+
+The SleepBuddy is a smart clock ⏰ that helps you sleep. It puts some calm music 🎵 and a night light 🔅 when you go to bed. The night light brightness and the music volume turns down as you fall asleep.
+
+The SleepBuddy can also :
+
+- Give the room temperature 🌡️
+- Turn on light (night light) when it detects something moving 🐕🐈🚶
+- Display 👀 the music you are listening to and the time thanks to its screens.
+- Share information about your sleep with your friend's SleepBuddy (cross project with @CodingForAddicts) thanks to its wifi 🛜 antenna.
+
+## ✅ Conditions to submit proposal:
+
+- [x] Be a current high school student 🧑‍🎓 (or younger)
+- [x] Submit an RSVP at https://hackclub.com/winter
+- [x] Be a member of the Hack Club Slack (https://hackclub.com/slack)
+- [x] Review my project proposal
+
+## 📃 Plan:
+1. Buy the hardware 💰
+2. Print the prototype of the box in a local FabLab
+3. Create the final box in polypropylene and wood
+4. Program 💻 my arduino with all the components (uses in the summary)
```

Comment:
Could you list out the components and their uses either here, or in the summary? Currently, you just have the uses listed out in the summary but not the components. (eg. use a temperature/humidity sensor to take the room temperature)

Reply:
I've added the components as explanations to the SleepBuddy's fonctions in the summary. Let me know if it's ok.

Reply:
Looks great, thanks!